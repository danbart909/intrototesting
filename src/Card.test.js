import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';

const card = {id: '1', title: 'anything', content: 'justastring'}

describe('Messages component', () => {

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Card
										key={card.id}
										title={card.title}
										content={card.content}
									/>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('this card should render', () => {
const tree = renderer
  .create(<Card
            key={card.id}
            title={card.title}
            content={card.content}
          />)
  .toJSON();
expect(tree).toMatchSnapshot();
});

})