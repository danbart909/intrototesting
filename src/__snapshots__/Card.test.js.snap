// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Messages component this card should render 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    anything
  </h3>
  <p>
    justastring
  </p>
</div>
`;
